pkg/exchange/batch/batch.go

```diff
@@ -95,7 +100,9 @@ func (e KLineBatchQuery) Query(ctx context.Context, symbol string, interval type
 			if len(kLines) == 0 {
 				return
 			}
+			const BatchSize = 200
 
+			var batchKLines = make([]types.KLine, 0, (len(kLines)/BatchSize)+1)
```

Comment:
since you will reset batchKLines when len == BatchSize, so the capacity should be BatchSize ?